andela/gondor-rc
# #159520464 Reject Deliveries

#### What does this PR do?
Add feature for customer to reject deliveries
#### Description of Task to be completed?
- create a reject delivery button on order history
- display a modal with textarea for reason for the rejection
- reject delivery button will only be available when vendor/admin ship the order
#### How should this be manually tested?
- login as a customer
- make order on any product of your choice and checkout
- choose pay on delivery for the method of payment
- login as a vendor
- ship the order
- the customer should visit the order history page
- click on the reject delivery button available at the top of the order details
- a modal will pop to enter the reason for rejection
- the vendor will see that the order as been rejected with the reason for rejection
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#159520464
#### Screenshots or gifs (if appropriate):
<img width="1438" alt="screen shot 2018-09-01 at 12 12 29 am" src="https://user-images.githubusercontent.com/31155397/44939351-606b6100-ad7c-11e8-9e7c-a8ee92d345cc.png">
<img width="1040" alt="screen shot 2018-09-01 at 12 12 44 am" src="https://user-images.githubusercontent.com/31155397/44939352-606b6100-ad7c-11e8-9074-6341f54f9ca0.png">
<img width="405" alt="screen shot 2018-09-01 at 12 16 46 am" src="https://user-images.githubusercontent.com/31155397/44939353-606b6100-ad7c-11e8-83cd-db95b0f70b31.png">

#### Questions:
